ctapipe/utils/index_finder.py

```diff
@@ -10,7 +10,7 @@ class IndexFinder:
     Helper class to find the index of the closest matching value in an array/list/...,
     used to locate the pointing of an event based on the trigger time.
     This searches using pythons bisect module.
-    All entries of `values` need to be unique.
+    Duplicated keys will be ignored.
```

Comment:
Values rather than keys?

Also I would add an warning here to avoid hard to find bugs if someone uses this class for other reasons